src/libraries/System.Reflection.Primitives/ref/System.Reflection.Primitives.cs

```diff
@@ -33,6 +33,7 @@ public enum FlowControl
         public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] object? obj) { throw null; }
         public bool Equals(System.Reflection.Emit.OpCode obj) { throw null; }
         public override int GetHashCode() { throw null; }
+        public int EvaluationStackDelta { get { throw null; } }
```

Comment:
Is this order produced by the ref assembly generator?
 
If you have not done it already, you may want to regenerate this ref assembly source (https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md).